src/main/java/core/basesyntax/model/Figure.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.model;
+
+public abstract class Figure {
+
+    public double area;
+    public Color color;
+
+    public abstract String draw();
```

Comment:
this method better to move to the interface:
- interface - behaviour
- abstract class - state